projects/igniteui-angular/src/lib/services/csv/csv-exporter-options.ts

```diff
@@ -29,18 +32,46 @@ export class IgxCsvExporterOptions extends IgxExporterOptionsBase {
         return extension;
     }
 
+    /**
+     * Gets the value delimiter which will be used for the exporting operation.
+     * ```typescript
+     * let delimiter = this.exportOptions.valueDelimiter;
+     * ```
+     * @memberof IgxCsvExporterOptions
+     */
     get valueDelimiter() {
         return this._valueDelimiter;
     }
 
+    /**
+     * Sets a value delimiter which will overwrite the default delimiter of the selected export format.
+     * ```typescript
+     * this.exportOptions.valueDelimiter = '|';
+     * ```
+     * @memberof IgxCsvExporterOptions
+     */
     set valueDelimiter(value) {
         this.setDelimiter(value);
     }
 
+    /**
+     * Gets the CSV export format.
+     * ```typescript
+     * let filetype = this.exportOptions.fileType;
+     * ```
+     * @memberof IgxCsvExporterOptions
+     */
     get fileType() {
         return this._fileType;
     }
 
+    /**
+     * Sets the CSV export format.
+     * ```typescript
+     * this.exportOptions.fileType = 'exp';
```

Comment:
**this.exportOptions.fileType** should be set to a CsvFileTypes enum value

Reply:
Fixed